src/math/Vector2.js

```diff
@@ -290,6 +290,28 @@ THREE.Vector2.prototype = {
 
 	}(),
 
+	clampLength: function () {
+
+		var oldLength, newLength;
```

Comment:
I think you do not need to put these variables into a closure as they do not hold an object that is re-used across invocations but just float value.